integration-test/src/test/java/org/bitcoinj/examples/ForwardingServiceTest.java

```diff
@@ -46,7 +46,7 @@ void setupTest() {
     public void startViaStaticForwardAndImmediatelyInterrupt(@TempDir File tempDir) {
```

Comment:
The test name reads like it now needs to be changed, too?

Reply:
Fixed and force-pushed.